### PR TITLE
version bump

### PR DESCRIPTION
**Pull request by CuriousVini**

### CODE REVIEW

**Review by sumengwang — APPROVED** [review]

File: integration-test-core/pom.xml [hunk 1]
```diff
@@ -21,7 +21,7 @@
   <parent>
     <artifactId>integration-test</artifactId>
     <groupId>io.cdap.cdap.test</groupId>
-    <version>1.0.0</version>
+    <version>6.10.0-SNAPSHOT</version>
```
Comment (itsankit-google) [inline integration-test-core/pom.xml]: Question: Seems like this dependency was not changed from `1.0.0` why this change is needed now??

**Review by itsankit-google — APPROVED** [review]
